prometheus_client/registry.py

```diff
@@ -128,7 +129,7 @@ def _target_info_metric(self):
         m.add_sample('target_info', self._target_info, 1)
         return m
 
-    def get_sample_value(self, name: str, labels: Optional[Dict[str, str]] = None) -> Optional[float]:
+    def get_sample_value(self, name: str, labels: Optional[Dict[str, str]] = None) -> Optional[Union[float, NativeHistStructValue]]:
```

Comment:
Does this need to change yet? It would be nice to have no/minimal public interface changes since we won't be using these in the registry quite yet.

Reply:
Same as for the Graphite bridge file. `mypy` linting fails otherwise.

Reply:
Solved after separating fields